### PR TITLE
[SIEM][Docs] Adds ml tuning task and new ml jobs

### PR DESCRIPTION
**Pull request by benskelker**
Adds the ML tuning task and new jobs for 7.5.

[Preview](http://stack-docs_666.docs-preview.app.elstc.co/guide/en/siem/guide/master/machine-learning.html)

### CODE REVIEW

File: docs/en/siem/machine-learning.asciidoc [hunk 1]
```diff
@@ -41,7 +41,24 @@ the time they are enabled. After jobs are enabled, they continuously analyse inc
...
+which beats are required on hosts for each job. For information on tuning anomaly results
+to reduce the number of false positive, see <<tuning-anomaly-results>>.
+
+[float]
+[[view-anomolies]]
+=== View detected anomalies
+To view the `Anomalies` table widget and `Max Anomaly Score By Job` details,
+the user must have the `ml_admin` or `ml_user` role.
+
+NOTE: To adjust the `score` threshold for which {xpack-ref}/xpack-ml.html[anomalies]
```
Comment (lcawl) [inline docs/en/siem/machine-learning.asciidoc]:
```suggestion
NOTE: To adjust the `score` threshold for which {stack-ov}/xpack-ml.html[anomalies]
```

File: docs/en/siem/machine-learning.asciidoc [hunk 2]
```diff
@@ -28,9 +28,9 @@ the job's `Group` field ({kib} -> {ml-cap} -> Create/Edit job -> Job details).
 
 [float]
 [[included-jobs]]
-==== Prebuilt Jobs
+==== Prebuilt jobs
 
-The SIEM app ships with prebuilt Machine Learning Jobs for detecting anomalies.
+The SIEM app ships with prebuilt Machine Learning jobs for detecting anomalies.
```
[the remark this answers is not in the document]
Comment (benskelker) [reply docs/en/siem/machine-learning.asciidoc]: Thanks @lcawl 
At the moment, we only use anomaly detection jobs. Which att should I use?

**Comment by randomuserid** [comment]
It's kind of hard to review here but we edited and reviewed the content in a gdoc and it looks good to me.

**Comment by benskelker** [comment]
@randomuserid - if it's easier you use the [preview](http://stack-docs_666.docs-preview.app.elstc.co/guide/en/siem/guide/master/machine-learning.html) next time. I always add the link to the first comment.

File: docs/en/siem/machine-learning.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (randomuserid) [reply docs/en/siem/machine-learning.asciidoc]: we have no data frames yet, so anomaly detection jobs, I think.

**Comment by randomuserid** [comment]
> @randomuserid - if it's easier you can use the [preview](http://stack-docs_666.docs-preview.app.elstc.co/guide/en/siem/guide/master/machine-learning.html) next time. I always add the link to the first comment.

oh right, thanks

File: docs/en/siem/machine-learning.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (lcawl) [reply docs/en/siem/machine-learning.asciidoc]: These are the pertinent attributes then: {anomaly-job}, {anomaly-jobs}, {anomaly-jobs-cap}.  Sometimes we throw a {ml} qualifier in front of those too.  If you want any additional input on any specific uses, @szabosteve or I are happy to help!

File: docs/en/siem/machine-learning.asciidoc [hunk 3]
```diff
@@ -28,10 +28,11 @@ the job's `Group` field ({kib} -> {ml-cap} -> Create/Edit job -> Job details).
...
+==== Prebuilt jobs
 
-The SIEM app ships with prebuilt Machine Learning Jobs for detecting anomalies.
-If you ship data using https://www.elastic.co/products/beats[Beats] and your environment
+The SIEM app ships with prebuilt {ml} {anomaly-jobs} for
+automatically detecting host and network anomalies. If you ship data using
+https://www.elastic.co/products/beats[Beats] and your environment
 is configured with the appropriate indices (`auditbeat-*`, `filebeat-*`, `packetbeat-*`,
 or `winlogbeat-*`) via {kib} -> Management -> Index Patterns, the jobs will be installed
 on page load, and will be displayed within the `Anomaly Detection` interface.
```
Comment (spong) [inline docs/en/siem/machine-learning.asciidoc]: For 7.5, as of https://github.com/elastic/kibana/pull/48067 this is no longer the case. All jobs will now be displayed by default/regardless of indices configured. Installation still happens in the background, but only when a user _enables_ a given job do we attempt to install the corresponding ML module. I don't think this aspect (installing) needs highlighted as it should make no difference to the user whether the job is _installed_ or _not_, just whether it is running or not.

**Comment by benskelker** [comment]
Say thanks to @randomuserid,:
> I wasn't aware of all the functionality under `Optimizing anomaly results`, so this was really informative -- thanks @benskelker! 🙂

Thanks for reviewing @spong - corrected.

**Pull request merged**